### PR TITLE
Storybook: add Jetpack Spinner

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request:

* Add existing component to Storybook.
* Add `color` param to allow overriding the color easily.
* Add tests.
* Change the default color to black.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions for Storybook:

* Go to the Storybook folder `cd projects/js-packages/storybook`
* Run `pnpm install`
* Run `pnpm run storybook:dev`
* Storybook should open in your browser.
* Go to Playground → Jetpack Spinner → Docs and play around with the spinner.

#### Screenshot

image

### CODE REVIEW

**Review by sergeymitr — CHANGES_REQUESTED** [review]
Great job, thank you!
I added a couple of minor suggestions, please take a look.

**Comment by keoshi** [comment]
@sergeymitr Beautiful suggestions, thanks for those. Pushed some changes and would be cool if you can take another look. Thank you!

**Comment by fgiannar** [comment]
@keoshi Looks great! Thanks for working on this!

I merged master and fixed the failed **Changelogger validity** tests, hope that's ok 😄 

Leaving it to @sergeymitr for a final review.

File: projects/js-packages/components/components/spinner/stories/index.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export default {
+	title: 'Playground/Spinner',
+	component: Spinner,
+};
+
+const Template = args => <Spinner { ...args } />;
+
+const DefaultArgs = {
+	moduleName: 'The Module Name',
```
Comment (sergeymitr) [inline projects/js-packages/components/components/spinner/stories/index.jsx:20]: This seems to be a leftover from the Footer component, I think we should remove it.

**Review by sergeymitr — DISMISSED** [review]
I noticed one minor thing, other than that everything looks awesome and ready to go!

**Comment by leogermani** [comment]
Ok, I went ahead and did a couple of things:

* Merged master and resolved conflicts
* removed unused argument (Sergey request)
* Added an option to use a color picker as a control for the color: 

Captura de Tela 2021-11-04 às 16 22 16

* Reverted the default color to white

This PR was changing the default color of the Spinner from white to black. This resulted in changes in the current UI:

Captura de Tela 2021-11-04 às 16 32 21

I reverted it to white and changed the default background color of the Storybook story to a dark one so the spinner is still visible.

I would give a green light to this PR right away but feels like cheating, so please have a look - and make sure to test the spinner in action in the Connect Screen.

**Review by sergeymitr — APPROVED** [review]
Looks great!
@keoshi, @leogermani thanks for making the changes!

**Comment by keoshi** [comment]
Thank you for pushing this across the finish line, @leogermani !

**Comment by leogermani** [comment]
r234600-wpcom

**Pull request merged**